spec/support/helpers/reporting.rb

```diff
@@ -1,4 +1,4 @@
-module SpecHelper
+module SpecHelpers
   module Reporting
```

Comment:
This is a different `Reporting`-module. It only includes some code from active support to capture a stream like `$stderr` or `$stdout` from code run in its block.